src/DssSpell.sol

```diff
@@ -50,90 +86,215 @@ contract DssSpellAction is DssAction {
     //
     // uint256 internal constant X_PCT_RATE      = ;
 
-    address internal constant D3M_HUB                   = 0x79Dcb858D6af6FeD7A5AC9B189ea14bC94076dfb;
-    address internal constant D3M_MOM                   = 0x8aBafFe006205e306F4307EE7b839846CD1ff471;
+    uint256 internal constant WAD = 10 ** 18;
+    uint256 internal constant MILLION = 10 ** 6;
 
-    address internal constant SPARK_D3M_PLAN            = 0x1fB2cF94D896bB50A17dD1Abd901172F088dF309;
-    address internal constant SPARK_D3M_POOL            = 0x8b6Ae79852bcae012CBc2244e4ef85c61BAeCE35;
-    address internal constant SPARK_D3M_ORACLE          = 0xa07C4eDB18E4B3cfB9B94D9CD348BbF6d5a7f4c2;
+    // -- RWA014 MIP21 components --
+    address internal constant RWA014                         = 0x22a7440DCfF0E8881Ec93cE519c34C15feB2A09a;
+    address internal constant MCD_JOIN_RWA014_A              = 0xc7Ba0aBa8512199c816834351CC978cf684D7fD9;
+    address internal constant RWA014_A_URN                   = 0xb475F63163aE3b0D5f6e30Dd914F5aA7204B1169;
+    address internal constant RWA014_A_JAR                   = 0x398E36Ed3c6bEf85f78b03d08b1980c6c3dd5357;
+    address internal constant RWA014_A_OUTPUT_CONDUIT        = 0x563c3CD928DB7cAf5B9872bFa2dd0E4F31158256;
+    address internal constant RWA014_A_INPUT_CONDUIT_URN     = 0x3b749869f62694804B0411DA77F13e816C49A25F;
+    address internal constant RWA014_A_INPUT_CONDUIT_JAR     = 0xa9C909eDD4ee06D625EaDD546CccDB1BB3e02D02;
+    // TODO: IPFS link
+    string  internal constant RWA014_DOC                     = "TODO";
+    uint256 internal constant RWA014_A_INITIAL_PRICE         = 500_000_000 * WAD;
+    uint48  internal constant RWA014_A_TAU                   = 0;
+    // Ilk registry params
+    uint256 internal constant RWA014_REG_CLASS_RWA           = 3;
+    // Remaining params
+    uint256 internal constant RWA014_A_LINE                  = 500_000_000;
+    uint256 internal constant RWA014_A_MAT                   = 100_00;
+    // Operator address
+    address internal constant RWA014_A_OPERATOR              = address(0); // TODO
+    // Custody address
+    address internal constant RWA014_A_COINBASE_CUSTODY      = address(0); // TODO
+    // -- RWA014 END --
 
-    address internal constant SPARK_ADAI                = 0x4480b29AB7a1b0761e6d0d480325de28B7266d73;
-    address internal constant SPARK_DAI_STABLE_DEBT     = 0xD72630D78157E1a2feD7A329873Bfd496704403D;
-    address internal constant SPARK_DAI_VARIABLE_DEBT   = 0xa99d874d26BdfD94d474Aa04f4f7861DCD55Cbf4;
-    address internal constant SPARK_POOL_CONFIGURATOR   = 0xe0C7ec61cC47e7c02b9B24F03f75C7BC406CCA98;
+    uint256 internal constant ZERO_SEVENTY_FIVE_PCT_RATE     = 1000000000236936036262880196;
```

Comment:
Let's discuss naming on a call so we can choose this as it's done here or `ZERO_PT_SEVEN_FIVE_PCT_RATE` as the future standard.

Reply:
Done